app.js

```diff
@@ -2,37 +2,32 @@
 
 console.log('Launching…');
 
+var exec = require('child_process').exec;
 var meta = require('./package.json');
 var express = require('express');
 var compression = require('compression');
 var bodyParser = require('body-parser');
-var exec = require('child_process').exec;
 var path = require('path');
 var Fs = require('fs');
-var Promise = require('promise');
+var Map = require('immutable').Map;
 var Uuid = require('node-uuid');
 
-var DocumentDownloader = require('./lib/document-downloader');
-var History = require('./lib/history');
-var JsonHttpService = require('./lib/json-http-service');
-var Publisher = require('./lib/publisher');
+var Job = require('./lib/job');
+var Orchestrator = require('./lib/orchestrator');
+var RequestState = require('./lib/request-state');
 var SpecberusWrapper = require('./functions.js').SpecberusWrapper;
-var ThirdPartyResourcesChecker = require('./lib/third-party-resources-checker');
-var TokenChecker = require('./functions.js').TokenChecker;
 
 // Configuration file
 require('./config.js');
 
 // Pseudo-constants:
 var STATUS_STARTED = 'started';
-var STATUS_ERROR = 'error';
-var STATUS_SUCCESS = 'success';
 
 var app = express();
 var requests = {};
+var port = process.argv[4] || global.DEFAULT_PORT;
```

Comment:
More generally (but this is completely for another PR), once you start having `process.argv[5]` I would suggest starting to use a CLI options parser (typically `nopt`). It can get painful to remember these things :)